core/src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -552,11 +552,14 @@ private Future<Void> categorizeNodes(List<Node> batch, String sourceKey, String
             node = Util.rebind(innerTx, node);
             Object value = node.getProperty(sourceKey, null);
             if (value != null) {
+                String nodeLabel = Util.sanitizeBackTicks(label);
+                String key = Util.sanitizeBackTicks(targetKey);
+                String relType = Util.sanitizeBackTicks(relationshipType);
```

Comment:
From apoc.refactor.categorize it looks like `label` could be null. I didn't check the others. You may need to handle this here, or in your sanitize function to avoid NPEs

Reply:
There seems to be other functions for which this might an issue.